code/game/objects/items/weapons/material/twohanded.dm

```diff
@@ -140,4 +141,5 @@
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	attack_verb = list("attacked", "poked", "jabbed", "torn", "gored")
 	default_material = "glass"
-	applies_material_colour = 0
\ No newline at end of file
+	applies_material_colour = 0
+	fragile = 1	//It's a haphazard thing of glass, wire, and steel
```

Comment:
Spear blades can be made out of almost any material, not just glass. You can apply steel blades, plasteel, diamond, uranium, etc.